modules/fluid_properties/include/userobjects/TabulatedFluidProperties.h

```diff
@@ -103,12 +105,31 @@ class TabulatedFluidProperties : public SinglePhaseFluidProperties
 
   virtual void rho_from_p_T(
       Real pressure, Real temperature, Real & rho, Real & drho_dp, Real & drho_dT) const override;
+  virtual void rho_from_p_T(const ADReal & pressure,
+                            const ADReal & temperature,
+                            ADReal & rho,
+                            ADReal & drho_dp,
+                            ADReal & drho_dT) const override;
```

Comment:
Are people using these interfaces with the AD derivatives?

Reply:
good point, probably not. Let s delete it for now

Reply:
they do. 
see in NSFV:
```
  addFunctorProperty<ADReal>(
      NS::time_deriv(NS::density),
      [this](const auto & r, const auto & t) -> ADReal
      {
        ADReal rho, drho_dp, drho_dT;
        _fluid.rho_from_p_T(_pressure(r, t), _temperature(r, t), rho, drho_dp, drho_dT);
        return drho_dp * _pressure.dot(r, t) + drho_dT * _temperature.dot(r, t);
      });
```